### PR TITLE
Article Teaser Molecule 

### LINKED ISSUE #32: Article Teaser Molecule
## Article Teaser

Large and small devices design
Screenshot 2019-06-28 at 12 06 46

Medium size devices layout design
Screenshot 2019-07-03 at 10 17 29

```
<ArticleTeaser data={data}>
 <Picture>
 <Text>
 <Text tag="h3">
</ArticleTeaser>
```

### PR DESCRIPTION
**Pull request by LouiaR**
Type: patch

Fixes #32

### CODE REVIEW

File: src/components/Molecules/ArticleTeaser/ArticleTeaser.test.js [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
       href="/test"
-      target="blank"
       images={images}
-      date=" 2 July 2019"
-      uppercase
-      weight="bold"
-      copy="News article"
+      date=" 01 July 2019"
+      title="News article"
       objectFit="cover"
```
Comment (gusliedke) [inline src/components/Molecules/ArticleTeaser/ArticleTeaser.test.js]: We can remove objectFit from here

File: src/components/Molecules/ArticleTeaser/ArticleTeaser.md [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
   href="/test"
-  target="blank"
   images={images}
-  date=" 2 July 2019"
-  uppercase={true}
-  weight="bold"
-  copy="News article"
+  date=" 01 July 2019"
+  title="News article"
   objectFit="cover"
```
Comment (gusliedke) [inline src/components/Molecules/ArticleTeaser/ArticleTeaser.md]: We can remove objectFit from here

Comment (LouiaR) [reply src/components/Molecules/ArticleTeaser/ArticleTeaser.md]: You want it to always be cover

**Comment by gusliedke** [comment]
Beautiful 🚀

**Pull request merged**